docs/architecture/adr-052-tendermint-mode.md

```diff
@@ -63,7 +63,7 @@ We would like to suggest a simple Tendermint mode abstraction. These modes will
 
 ## Status
 
-Proposed
+Accepted
```

Comment:
I think this has been Implemented. At least I thought I implemented it. There may be things such as revising the config that need to be done but I would consider it implemented

Reply:
whoops, didnt hit save on that file